dashboard/cron.py

```diff
@@ -80,6 +80,37 @@ class DashboardCronJob(CronJobBase):
     schedule = Schedule(run_at_times=settings.RUN_AT_TIMES)
     code = 'dashboard.DashboardCronJob'    # a unique code
 
+    # return a list of invalid course ids
+    def invalid_course_ids(self):
+
+        # whether all course ids are valid ids
+        invalid_course_id_list = []
+
+        logger.debug("in checking course")
+
+        # loop through multiple course ids
+        for course_id in Course.objects.get_supported_courses():
+            if (not course_id.isdigit()):
```

Comment:
```suggestion
            if not course_id.isdigit():
```

Reply:
There's not much in the style guide about if statements. I believe either form is _fine_ but best practice is for single line if statement to **not** have parenthesis but multiple lines should, so looks good.

https://www.python.org/dev/peps/pep-0008/#multiline-if-statements

Reply:
parentheses removed.